### PR TITLE
todo

### PR DESCRIPTION
**Pull request by Rebecza**
**What problem is the PR solving / What's new?**
A description of the problem that is being solved, or which new feature is implemented. Please link to other relevant issues/pr if relevant

**What did change**
Explain clearly and concisely what you changed. Why did you take certain design decisions?

**Checklist**
- [x] I made a PR to develop (not master)
- [ ] If applicable: I updated the docs
- [ ] I updated the CHANGELOG
- [ ] These changes are covered by the tests

### CODE REVIEW

**Comment by Rebecza** [comment]
Do not look at the position of to do-comment in first commit: should've been computeMatrix_gene (instead of computeMatrix_peaks).

Latest commit: Now generates a plotProfile_gene of reads distribution around the gene like this: 
afbeelding

File: seq2science/rules/qc.smk [hunk 1]
```diff
@@ -329,10 +329,11 @@ rule computeMatrix_gene:
         if get_descriptive_names(wildcards, input.bw) != ""
         else "",
         gtf=rules.get_genome_annotation.output.gtf,  # TODO: move genomepy to checkpoint and this as input
+        params=config["deeptools_computematrix_gene"],
     shell:
         """
         computeMatrix scale-regions -S {input.bw} {params.labels} -R {params.gtf} \
-        -p {threads} -b 2000 -a 500 -o {output} > {log} 2>&1
+        -p {threads} --beforeRegionStartLength 3000 --regionBodyLength 5000 --afterRegionStartLength 3000 -o {output} > {log} 2>&1
```
Comment (Maarten-vd-Sande) [inline seq2science/rules/qc.smk:336]: you should replace the `--beforeRegionStartLength 3000 --regionBodyLength 5000 --afterRegionStartLength 3000` with `{params.params}`. This way the actual `deeptools_computematrix_gene` config option is used 😄

Comment (Rebecza) [reply seq2science/rules/qc.smk:336]: ooooh! 🧐 ...on it!

**Review by Maarten-vd-Sande — APPROVED** [review]
Awesome :tada:! Can be merged :smile:

**Comment by Rebecza** [comment]
🎉

**Pull request merged**